README.md

```diff
@@ -54,6 +54,22 @@ a different API server, copy `app/config.js` to
 not tracked and will be used instead if it exists.
 
 #### Before opening a pull request
+
+Please configure your editor to using the
+following settings to avoid common code inconsistencies and dirty
+diffs:
+
+* Use soft-tabs set to two spaces.
+* Trim trailing white space on save.
```

Comment:
Not saying we shouldn't, but it will create a lot of noisy diffs until the trailing space is gone from all of our files.